api/src/main/java/io/strimzi/api/kafka/model/KafkaExternalBrokerService.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Configures external broker service and advertised addresses
+ */
+@JsonPropertyOrder({"broker", "advertisedHost", "advertisedPort", "nodePort"})
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Buildable(
+    editableEnabled = false,
+    generateBuilderPackage = false,
+    builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+public class KafkaExternalBrokerService implements Serializable {
+    private static final long serialVersionUID = -7537621108590168627L;
+
+    private Integer broker;
+    private String advertisedHost;
+    private Integer advertisedPort;
+    private Integer nodePort;
+
+    public KafkaExternalBrokerService() {
+    }
+
+    public KafkaExternalBrokerService(Integer broker, String advertisedHost, Integer advertisedPort, Integer nodePort) {
```

Comment:
We see the same thing here about constructors. To me there's little benefit to having a constructor like this when the `@Buildable` annotation means we will have a `KafkaExternalBrokerServiceBuilder` generator which is usually more helpful when using the `KafkaExternalBrokerService` in a bigger tree of objects.